frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -105,13 +105,14 @@ const EmailTemplate = ({
       <SaveDraftModal
         saveable
         onSave={async () => {
-          if (campaignId && subject && body && from) {
-            try {
-              await saveTemplate(+campaignId, subject, body, replyTo, from)
-            } catch (err) {
-              setErrorMsg(err.message)
-              throw err
-            }
+          if (!campaignId) return
+          try {
+            if (!subject || !body)
+              throw new Error('Message subject or template cannot be empty!')
```

Comment:
Technically, we don't need to manually throw an error because the backend endpoint would throw an error if either body or subject is empty. 

But with that said, I think this error message is more user friendly compared to the one from the backend endpoint. 

![image](https://user-images.githubusercontent.com/3666479/101734364-1c3e9d80-3afb-11eb-811e-36be6c0a815b.png)